src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: Props) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
Well, you have already found a selectedTab at the `App`, you should have passed it as a prop. 
After adjustments there will no need at all at finding a tab by id.